### DISCUSSION TITLE
How to fuse an unknown number of inputs?

### DISCUSSION THREAD

**Discussion by BarbourSmith** [sgenoud/replicad#76] [How To]
Hello!

Is there a way to fuse an array of shapes all at once? 

cad.fuse(...inputs) or something?

No worries if there isn't I can write my own, just wondering if I'm missing a built in way to do it.

**Comment by BarbourSmith** [comment]
I'm going to delete this because I can't find any reference to another way to do it, and writing one was trivial

**Comment by sgenoud** [comment]
There is no built-in support - because, as you noted, this is fairly trivial, and can have different needs (i.e. what does a cutAll do?).

I should add a recipe for that as it is a common use case.

**Reply by sgenoud** [reply]
I have added it here: https://replicad.xyz/docs/recipes/fuse-all
